dev/ci/gitlab.bat

```diff
@@ -28,11 +31,35 @@ if exist %DESTCOQ%\ rd /s /q %DESTCOQ%
 call %CI_PROJECT_DIR%\dev\build\windows\MakeCoq_MinGW.bat -threads=1 ^
   -arch=%ARCH% -installer=Y -coqver=%CI_PROJECT_DIR_CFMT% ^
   -destcyg=%CYGROOT% -destcoq=%DESTCOQ% -cygcache=%CYGCACHE% ^
-  -addon="bignums ltac2 equations" -make=N ^
-  -setup %CI_PROJECT_DIR%\%SETUP% || GOTO ErrorExit
+  -addon=bignums ^
+  -addon=equations ^
+  -addon=ltac2 ^
+  -addon=mathcomp ^
+  -addon=menhir ^
+  -addon=menhirlib ^
+  -addon=compcert ^
+  -addon=vst ^
+  -addon=aactactics ^
+  -addon=quickchick ^
+  -addon=coquelicot ^
```

Comment:
ext-lib is not listed here. Is it an oversight?

Reply:
Yes. It is installed as part of QuickChick, but it should be a separate component.

Btw.: There is no dependency handling in there as yet - not because it would be difficult to handle on the build script side, but NSIS doesn't have this build in - all manual programming.

I thought about at least adding comments "Please also select this or that". I could also fix it, but it would take me half a day.

Regarding MTAC-2: yes this got lost in all the fuzz. If you can easily do it, please go ahead. I can also do it - would be ready on Monday.

Reply:
@MSoegtropIMC I re-opened #8305. Regarding your remark on dependency handling, this seems serious enough. It should not blow up on users if they select only one package and not its dependencies.

Reply:
@Zimmi48 : how do we solve the installer dependency thing? Just put comments in the package description or do it properly? I could do it properly over the weekend.

Reply:
I prefer to do it properly, even if it means delaying 8.8.2 by a couple more days.